app-core/src/main/java/net/spals/appbuilder/app/core/generic/GenericWorkerApp.java

```diff
@@ -105,6 +124,7 @@ public Builder setServiceScan(final ServiceScan serviceScan) {
             addBootstrapModule(new AutoBindModulesBootstrapModule(serviceScan));
             // 3. Use the serviceScan to find auto bound services
             servicesModuleBuilder.setServiceScan(serviceScan);
+            servicesModuleIndex.set(orderedModules.size());
```

Comment:
So what is the index set in the scan? But build  is the one modifying orderedModules. Am I misunderstanding the interaction between build() and setServiceScan()?